provider/Cargo.toml

```diff
@@ -14,3 +14,7 @@ reqwest = { version = "0.10.4", features = ["blocking", "json"] }
 serde = "1.0.106"
 serde_json = "1.0.51"
 codec = { path = "../codec" }
+aws_lambda_events = { git = "https://github.com/ewbankkit/aws-lambda-events", branch = "include-aws-lambda-go.pr-280" }
```

Comment:
Until

* https://github.com/aws/aws-lambda-go/pull/280
* https://github.com/aws/aws-lambda-go/pull/285

are merged and https://github.com/LegNeato/aws-lambda-events is regenerated.